CONTRIBUTING.md

```diff
@@ -40,15 +40,15 @@ We welcome contributions to kURL. We appreciate your time and help.
       name: testing
     spec:
       kubernetes:
-        version: 1.19.3
+        version: 1.19.7
       weave:
         version: 2.7.0
       openebs:
         version: 1.12.0
         isLocalPVEnabled: true
         localPVStorageClassName: default
-      docker:
-        version: 19.03.10
+      containerd:
```

Comment:
I'd like to match the version used in the "how to download existing versions" and move towards containerd instead of docker.